operator/src/main/resources/Operator.properties

```diff
@@ -189,6 +189,7 @@ WLSKO-0176=Job {1} in namespace {0} failed, job details are {2}
 WLSKO-0177=Pod {0} in namespace {1} failed, the pod status is {2}
 WLSKO-0178=Operator cannot proceed, as the Custom Resource Definition for ''domains.weblogic.oracle'' is not installed.
 WLSKO-0179=Pod {0} in namespace {1} detected as stuck, and force-deleted
+WLSKO-0180=Creating event: {0}
```

Comment:
Does the even have details about the domain and namespace?

Reply:
The event resource are namespaced, and each event is associated with a domain resource.
As far as this log message is concerned, even operator log message contains the domainUID and the namespace that it is associated with.